sql/wsrep_var.cc

```diff
@@ -478,23 +478,24 @@ bool wsrep_cluster_address_update (sys_var *self, THD* thd, enum_var_type type)
      Note: releasing LOCK_global_system_variables may cause race condition, if 
      there can be several concurrent clients changing wsrep_provider
   */
+  WSREP_DEBUG("wsrep_cluster_address_update: %s", wsrep_cluster_address);
   mysql_mutex_unlock(&LOCK_global_system_variables);
   wsrep_stop_replication(thd);
 
-
   if (wsrep_start_replication())
   {
     wsrep_create_rollbacker();
     wsrep_create_appliers(wsrep_slave_threads);
   }
-
   /* locking order to be enforced is:
      1. LOCK_global_system_variables
      2. LOCK_wsrep_slave_threads
+     => have to juggle mutexes to comply with this
```

Comment:
Comment should say `LOCK_wsrep_cluster_config` instead of `LOCK_wsrep_slave_threads`.